### PR TITLE
Added few faqs

### PR DESCRIPTION
**Pull request by kartikeya2609**
Added few faqs

Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

PRs must meet these requirements to be merged:

- [ ] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

**Review by douglas-j-bothwell — DISMISSED** [review]
STO FAQs approved.

File: kb/continuous-delivery/continuous-delivery-faqs.md [hunk 1]
```diff
@@ -8061,3 +8061,10 @@ Here is an example expression that converts a string variable named myString to
 You can use this expression in any setting that supports expressions in Harness.
 
 
+#### Is the "Enable Native Helm steady state for jobs" option in default setting is for the steady state check?
```
Comment (SushrutHarness) [inline kb/continuous-delivery/continuous-delivery-faqs.md:8064]:
```suggestion
#### Is the "Enable Native Helm steady state for jobs" option a default setting for the steady state check?
```

File: kb/continuous-delivery/continuous-delivery-faqs.md [hunk 2]
```diff
@@ -8061,3 +8061,10 @@ Here is an example expression that converts a string variable named myString to
...
+This selection is solely for verifying the steady state of a Kubernetes Job deployed via the Helm chart.
+
+#### Is there a way to get the notification whenever the new pipeline gets created?
```
Comment (SushrutHarness) [inline kb/continuous-delivery/continuous-delivery-faqs.md:8067]:
```suggestion
#### Is there a way to get a notification whenever a new pipeline gets created?
```

**Review by SushrutHarness — COMMENTED** [review]
Typos

File: kb/continuous-delivery/continuous-delivery-faqs.md [hunk 3]
```diff
@@ -8061,3 +8061,10 @@ Here is an example expression that converts a string variable named myString to
 You can use this expression in any setting that supports expressions in Harness.
 
 
+#### Is the "Enable Native Helm steady state for jobs" option a default setting for the steady state check?
+This selection is solely for verifying the steady state of a Kubernetes Job deployed via the Helm chart.
+
+#### Is there a way to get a notification whenever a new pipeline gets created?
```
Comment (roshnisarangadharan) [inline kb/continuous-delivery/continuous-delivery-faqs.md:8067]:
```suggestion
#### Is there a way to get notified whenever a new pipeline is created?
```

**Review by roshnisarangadharan — DISMISSED** [review]

**Pull request merged**